src/cdb.js

```diff
@@ -1,36 +1,32 @@
 // Creates cdb object, mutated in the entry file cartodb.js
 // Used to avoid circular dependencies
-var cdb = window.cdb = window.cartodb;
+var cdb = {};
 
 // Might be set in different bundles (e.g. cartodb.js vs. torque), avoid overriding it once set
-if (!cdb) {
-  cdb = window.cdb = window.cartodb = {};
-  if (!window.JST) window.JST = {};
-  cdb.VERSION = "3.15.8";
-  cdb.DEBUG = false;
-  cdb.CARTOCSS_DEFAULT_VERSION = '2.1.1';
-  cdb.CARTOCSS_VERSIONS = {
-    '2.0.0': '',
-    '2.1.0': ''
-  };
+cdb.VERSION = require('../package.json').version;
+cdb.DEBUG = false;
+cdb.CARTOCSS_DEFAULT_VERSION = '2.1.1';
+cdb.CARTOCSS_VERSIONS = {
+  '2.0.0': '',
+  '2.1.0': ''
+};
 
-  cdb.config = {};
-  cdb.core = {};
-  cdb.decorators = {};
-  cdb.image = {};
+cdb.config = {};
+cdb.core = {};
+cdb.decorators = {};
+cdb.image = {};
 
-  cdb.geo = {};
-  cdb.geo.common = {};
-  cdb.geo.geocoder = {};
-  cdb.geo.ui = {};
+cdb.geo = {};
+cdb.geo.common = {};
+cdb.geo.geocoder = {};
+cdb.geo.ui = {};
 
-  cdb.ui = {};
-  cdb.ui.common = {};
+cdb.ui = {};
+cdb.ui.common = {};
 
-  cdb.vis = {};
+cdb.vis = {};
 
-  cdb.windshaft = {};
-  cdb.windshaft.filters = {};
-}
+cdb.windshaft = {};
+cdb.windshaft.filters = {};
```

Comment:
Not sure if we need these guys anymore. They were probably introduced before everything was browserified.